src/main/java/nextstep/subway/domain/Line.java

```diff
@@ -40,6 +41,10 @@ public void removeSection(Station station) {
         sections.removeSection(station);
     }
 
+    public List<Section> getSectionList() {
```

Comment:
메서드 네이밍 시 특정 자료형을 의미하는 부분을 제거해보면 어떨까요?
```suggestion
    public List<Section> getSections() {
```